zcash-hsmbuilder/Cargo.toml

```diff
@@ -41,7 +41,8 @@ rand_core = "0.6"
 ripemd = "0.1"
 secp256k1 = { version = "0.21" }
 sha2 = "0.9"
-
+log4rs = "1"
```

Comment:
unused - but should be used for binaries, not libraries (in that case `log` should be used)

Reply:
also `chrono` seems unused